test/cloudops.go

```diff
@@ -59,6 +59,12 @@ func compute(t *testing.T, driver cloudops.Ops) {
 	groupInfo, err := driver.InspectInstanceGroupForInstance(instanceID)
 	require.NoError(t, err, "failed to inspect instance group")
 	require.NotNil(t, groupInfo, "got nil instance group info from inspect")
+
+	err = driver.SetCountForInstanceGroup(groupInfo, int64(4))
+	require.NoError(t, err, "failed to inspect instance group")
```

Comment:
Ignore if the driver returns not supported error

Also the message is incorrect

Reply:
@harsh-px updated error msg.
Any idea how we can avoid assertion failures for only certain kind of errors here?